dashboard/app/helpers/ai_system_prompts/system_prompt_helper.rb

```diff
@@ -1,46 +1,25 @@
 require 'cdo/shared_constants'
 
 # Helper methods for generating system prompts for AI shared across
-# AI Tutor and Evaluate Student Learning.
+# AI Tutor v1 and Evaluate Student Learning.
 module AiSystemPrompts::SystemPromptHelper
-  def self.get_basic_system_prompt(level, unit)
+  def self.get_basic_system_prompt(level)
     base_prompt =
-      "You are an expert Computer Science teacher. Your students are in grades: #{get_grade_levels(unit)}. The programming language they are learning is #{get_programming_language(unit)}. The student is working on a level. #{get_level_instructions(level)}."
-    if programming_level?(level)
+      "You are an expert Computer Science teacher. Your students are in grades: #{level.grade_levels}. \n\n #{get_level_instructions(level)}"
+    if level.upper_grades_programming_level?
       base_prompt += get_starter_code(level)
       base_prompt += get_validated_level_test_file_contents(level)
     end
-    base_prompt
-  end
-
-  def self.get_grade_levels(unit)
-    unit.get_course_version.course_offering.grade_levels
-  end
-
-  # TODO: see if there's a better way to do this maybe via marketing info for course catalog?
-  # TODO: also see if there's a unit description that would be helpful. There's one for courses,
-  # but it might be too broad.
-  def self.get_programming_language(unit)
-    if unit.csa?
-      "Java"
-    elsif unit.csp?
-      "JavaScript"
-    else
-      "Python"
-    end
-  end
```

Comment:
I've found in playing around with evaluations that the AI doesn't need the programming language explicitly stated in the system prompt, but we can add in a method on the level model to determine language if we find we need it.